graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/TypeUtils.kt

```diff
@@ -112,6 +112,30 @@ class TypeUtils(private val packageName: String, private val config: CodeGenConf
         return NodeTraverser().postOrder(visitor, fieldType) as JavaTypeName
     }
 
+    /**
+     * Takes a GQL interface type name and returns the appropriate kotlin type given all of the mappings defined in the schema and config
+     */
+    fun findKtInterfaceName(interfaceName: String, packageName: String): JavaTypeName {
```

Comment:
nit: Consider changing the name of this method to findJavaInterfaceName

Reply:
Done!